docs/decision_record.md

```diff
@@ -8,9 +8,37 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 [TOC]
 
+## 2024-01-10 Clarifications on the purpose and form of the JSON schema in service-info
+
+### Decision
+
+We made a series of decisions regarding how the JSON-schema should be used to specify the data that a seqcol server will serve.
+
+- we RECOMMEND you produce an openapi.json for your service.
```

Comment:
I agree with the recommendation but I'm not sure it makes sense to have this decision recorded here. It feels a bit out of scope